public/site.webmanifest

```
@@ -0,0 +1,19 @@
+{
+  "name": "",
+  "short_name": "",
```

Should we add names here?
```suggestion
  "name": "Northeastern Class of 2025",
  "short_name": "NEU '25",
```